core/trino-main/src/test/java/io/trino/security/TestFileBasedSystemAccessControl.java

```diff
@@ -854,11 +855,11 @@ private String getResourcePath(String resourceName)
     public void parseUnknownRules()
     {
         assertThatThrownBy(() -> parse("src/test/resources/security-config-file-with-unknown-rules.json"))
-                .hasMessageContaining("Invalid JSON");
+                .hasMessageContaining("Invalid JSON file");
```

Comment:
Changes in the assertion messages can be extracted to a new commit or we can revert it.

Reply:
done